### ISSUE TITLE
Disable "Charecter Count", only display "Word Count" 🔠

### ISSUE THREAD

**Issue by mahadevan** [lukeleppan/better-word-count#56]
How to disable "Character Count", and only display "Word Count" ? 

There is no option for it in the plugin options. 😞

**Comment by lukeleppan** [comment]
Hi, thanks for the issue! I do appreciate everyone who takes the time to notify me of issues. I have good news.

Firstly, I would like to mention that I did stop work on the plugin after the last update, mainly do to the stress I was under from other commitments. I stopped shortly after releasing the current version of the plugin. As mentioned in the README, the plugin is in a very featureless state only replacing the default word count. No customisation is currently available. The reason for this was because of the obsidian update that added WYSIWYG. This update changed the way that the editor worked and so the old method of detecting text selection and reading text no longer worked. This, combined with some poorly written code on my end, meant the the plugin needed to be rewritten entirely.

Secondly, the good news, I have begun work on the plugin in my spare time again. A lot of progress has been made in the rewrite. All the features have been reimplemented and are written in a manner that I am much happier with. I am in the process of cleaning up and adding backwards compatibility for the legacy editor. Once that is complete I will release the update (probably next week but no promises 😉). I have seen many other issues suggesting features and I am open to implementing them slowly but surely.

To answer your question, you will be able to customise the counter to only display word count next week.

Final notes on plugin development, the plugin has a far better a customisation setting which is heavily inspired by the Breadcrumbs plugin's management of hierarchies. I want to mention how much I enjoy the process of developing plugins on obsidian. With the lack of documentation, it can be time consuming since lots of time has to go into research. To develop a plugin, one not only has to scan through the API but often also look at the way other plugins have been developed and adapt their techniques to develop a solution. Making it quite a collaborative process. I simply want to mention that the process is fascinating and beautiful.

As for Better Word Count, I have a vision for its future. I hope that it will one day be able to provide users with many different insights into their writing from statistics that are stored locally and not on the cloud. Obviously, this will all be optional and will have to be enabled in settings so that the default of the plugin is just a basic replacement to the default word count.

Thank you again, enjoy your Tuesday.
